seed/php-sdk/imdb/src/Core/JsonDecoder.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Seed\Core;
+
+use DateTime;
+use Exception;
+use JsonException;
+
+class JsonDecoder
+{
+    /**
+     * @throws JsonException
+     */
+    public static function decodeString(string $json): string {
+        $decoded = self::decode($json);
+        if (!is_string($decoded)) {
+            throw new JsonException("Unexpected non-string json value: " . $json);
+        }
+        return $decoded;
+    }
+
+    /**
+     * @throws JsonException
+     */
+    public static function decodeBoolean(string $json): bool {
+        $decoded = self::decode($json);
+        if (!is_bool($decoded)) {
+            throw new JsonException("Unexpected non-string json value: " . $json);
+        }
+        return $decoded;
+    }
+
+    /**
+     * @throws JsonException
+     */
+    public static function decodeDatetime(string $json): DateTime {
```

Comment:
```suggestion
    public static function decodeDateTime(string $json): DateTime {
```

^ I noticed one of your other functions uses the `DateTime` casing convention. I'm fine with either one as long as it's consistent.